gcloud/bigquery/table.py

```diff
@@ -22,6 +22,35 @@
 from gcloud.bigquery._helpers import _prop_from_datetime
 
 
+class SchemaField(object):
+    """Describe a single field within a table schema.
+
+    :type name: string
+    :param name: the name of the field
+
+    :type field_type: string
+    :param field_type: the type of the field (one of 'STRING', 'INTEGER',
+                       'FLOAT', 'BOOLEAN', 'TIMESTAMP' or 'RECORD')
```

Comment:
I don't know.  If we do pick one, then I think the PEP 435-compatible https://pypi.python.org/pypi/enum34/ is probably the way to go:  it ports the PEP, which landed in Python 3.4, back to all the versions we care about (and more).